.storybook/themes/news.scss

```
@@ -812,6 +812,53 @@
 					list-style-position: inside,
 					list-style-type: disc,
 				),
+				author-bio: (
+					gap: var(--global-spacing-4),
+				),
+				author-bio-author: (
+					gap: var(--global-spacing-4),
+					display: flex,
+					flex-direction: row,
+					components: (
+						image: (
+							max-width: var(--global-spacing-12),
+							max-height: var(--global-spacing-12),
+							border-radius: var(--global-border-radius-2),
+						),
+						stack: (
+							flex: 1,
+						),
+					),
+				),
+				author-bio-social-link-wrapper: (
+					display: flex,
+					flex-wrap: wrap,
+					components: (
+						link: (
+							margin: var(--global-spacing-2) var(--global-spacing-4) 0 0,
+						),
+					),
+				),
+				author-bio-social-link-hover: (
+					components: (
+						icon: (
+							fill: var(--color-primary-hover),
+						),
+					),
+				),
+				author-bio-authorName: (
+					font-family: var(--font-family-primary),
+					font-size: var(--global-font-size-6),
+					font-weight: var(--global-font-weight-7),
+					margin-bottom: var(--global-spacing-2),
+					line-height: var(--global-line-height-5),
+				),
+				author-bio-authorDescription: (
```

```suggestion
				author-bio-author-description: (
```
we shouldn't need to use camelCase here. In the past, we've used dash-separated token names as a style. Should we go back over the styling tokens documentations?

---

as much as I like camelCase... I agree: Convention is dashed.

---

addressed in [b380393](https://github.com/WPMedia/arc-themes-blocks/pull/1474/commits/b380393a9747325cb5ce5c8048ebad658e57155a)

---

fixed